api/src/controllers/support-center.js

```diff
@@ -161,9 +162,15 @@ router.post("/ticket", passport.authenticate(["referent", "young"], { session: f
     if (req.body.group && ZAMMAD_GROUP.includes(req.body.group)) {
       group = req.body.group;
     } else if (isYoung(req.user)) {
-      group = ZAMMAD_GROUP.YOUNG;
-    } else {
+      group = ZAMMAD_GROUP.VOLONTAIRE;
+    } else if (req.user.role === ROLES.REFERENT_DEPARTMENT || req.user.role === ROLES.REFERENT_REGION) {
       group = ZAMMAD_GROUP.REFERENT;
+    } else if (req.user.role === ROLES.ADMIN) {
+      group = ZAMMAD_GROUP.ADMIN;
+    } else if (req.user.role === ROLES.RESPONSIBLE) {
```

Comment:
`SUPERVISOR ` est l'équivalent de `RESPONSIBLE`, mais pour le tête de réseau
```suggestion
    } else if (req.user.role === ROLES.RESPONSIBLE || req.user.role === ROLES.SUPERVISOR) {
```